<h2 id="biomedical-data-analysis">Biomedical Data Analysis</h2>

<p>Our research interests include the analysis of different types of high dimensionality data of interest in Molecular Biology and in the Biomedical field. Examples of data types we have worked on include molecular profiles (mRNA, miRNA, proteins), time series (sensor data, clinical variables from longitudinal studies), usage data of smartphone apps for clinical use.  Another topic we are interested in is the inference and application of networks, particularly those commonly employed in Systems Biology to represent gene/protein interactions. Most of these lines of research are carried out in collaboration with the Microsoft Research - University of Trento Centre for Computational and Systems Biology (COSBI) in Rovereto.</p>

<h3 id="main-research-topics">Main research topics</h3>

<h5 id="rank-based-signatures-as-biomarkers-of-disease">Rank-based signatures as biomarkers of disease</h5>

<p>The classification of biological samples by means of their respective molecular profiles is a topic of great interest for its potential diagnostic, prognostic and investigational applications. We have proposed a method for the classification of molecular profiles based on a radically new approach consisting in the analysis of the similarity of rank-based sample-specific signatures. We have already used this method for a number of projects, but due to its novelty there is ample room for conceptual improvements and for exploring further applications.</p>

<h5 id="novel-methods-for-multi-omic-analysis">Novel methods for multi-omic analysis</h5>

<p>A current research challenge in molecular data analysis is how to integrate multiple data types for the same subjects in order to increase the chance of identifying relevant phenotype groups. In an ongoing project carried out in collaboration with COSBI and CIBIO we have developed a new method for multi-omics integration which is able to maintain good clustering performance across a wide range of omics data types. Using publicly available data we verified its efficiency in clustering patients based on survival for ten different cancer types.</p>

<h5 id="methods-for-the-inference-of-personalized-regulatory-networks">Methods for the inference of personalized regulatory networks</h5>

<p>Our research on gene regulatory network inference is focused on the development of a method based on structural equation modeling to infer condition-specific networks. Starting with one or possibly two groups of patients (i.e. healthy and affected) and their molecular profiles, our method exploits different types of data  (i.e. expression profile and genotype) in order to increase the accuracy of the regulatory network generated for the specific group(s) of patients. To further improve the quality of the inferred network, our approach takes advantage of the wealth of existing information about gene interactions and uses this knowledge to carefully inform the construction of the network.</p>

<h5 id="smartphone-applications-to-improve-patient-adherence-to-clinical-protocols">Smartphone applications to improve patient adherence to clinical protocols</h5>

<p>Mobile apps are an example of digital health tools with the potential to improve treatment outcome and patient wellness. The aim of a study conducted at a local hospital we were involved with was to assess whether a mobile application can increase compliance with an internationally developed pre/post-operative protocol for patients to follow to improve surgery-related outcomes and patient wellness. Using usage data collected on a cohort of 400+ patients, the study demonstrated the effectiveness of the app in increasing patient adherence to the protocol and identified factors that influenced its use for possible improvements to its design.</p>

<h5 id="clustering-of-temporal-profiles">Clustering of temporal profiles</h5>

<p>In longitudinal clinical studies, methodologies available for the analysis of multivariate data with multivariate methods are relatively limited. We have proposed a new computational method based on clustering of time profiles and posterior identification of correlation between clusters and predictors. In one application, we have tested its application to a clinical database which contains temporal variations of clinical, metabolic, and anthropometric profiles in a 100+ population of children followed-up annually for a period of ten years.</p>

<h5 id="analysis-of-data-from-wearable-sensors">Analysis of data from wearable sensors</h5>

<p>The ubiquitous nature of smartphones and their built-in sensors makes them an attractive tool for digital health applications. In one ongoing project we are studying methods for the analysis of accelerometer data collected on smartphones for applications such as diagnosis and monitoring of neurodegenerative diseases.</p>

<h3 id="collaborations">Collaborations</h3>

<p>Microsoft Research - University of Trento Centre for Computational and Systems Biology (COSBI), Rovereto
CIBIO Department,  Università di Trento - Enrico Domenici and Luca Marchetti
Ospedale Sacro Cuore Don Calabria, Negrar di Valpolicella (VR) - Massimo Guerriero and Elisa Bertocchi</p>

<h3 id="thesis-proposals">Thesis proposals.</h3>

<p>Please contact <a href="https://webapps.unitn.it/du/it/Persona/PER0160323/Curriculum">Prof. Lauria</a> if you are interested in research projects regarding possible topics for a master thesis or a PhD in applied mathematics/computational biology..</p>

<hr />

<h2 id="advanced-statistical-methods-for-complex-data-in-biomedicine">Advanced Statistical Methods for Complex Data in Biomedicine</h2>

<h3 id="description">Description</h3>

<p>We are interested in the development of statistical methodology and computational approaches related to graphical modelling of high dimensional multivariate data and modelling of network data. In the application area of biomedicine, we are particularly interested in the reconstruction of biological networks from high-throughput genomic data and in the modelling of the dynamic processes of cell differentiation and disease spreading.</p>

<h3 id="main-research-topics-1">Main research topics</h3>

<h5 id="regularized-inference-for-high-dimensional-data">Regularized inference for high-dimensional data</h5>

<p>We study the penalised inferential framework in the context of graphical modelling, that is for the estimation of the inverse of the covariance matrix. We develop efficient computational methods for solving the estimating equations, in a variety of (non-standard) settings such as non-Gaussian and missing/censored data.</p>

<h5 id="reconstruction-of-biological-networks">Reconstruction of biological networks</h5>

<p>We apply graphical modelling approaches to the inference of biological networks from high-throughput genomic data. We develop bespoke models that adapt to the richness, heterogeneity and complexity of genomic data and implement efficient frequentist and Bayesian inferential schemes for statistical inference. In recent years, we have conducted extensive research on the reconstructing of transcriptomic networks and microbiota systems.</p>

<h5 id="statistical-inference-of-quasi-reaction-systems">Statistical inference of quasi-reaction systems</h5>

<p>We develop statistical methods for the inference of parameters of dynamic processes, such as cell differentiation and disease spreading, that can be characterized via quasi-reaction systems of stochastic differential equations. Recently, we are particularly interested in developing methods that combine local linear approximation procedures with event-history analyses.</p>

<h3 id="main-collaborations">Main collaborations</h3>

<ul>
  <li>Institute of Computing, Università Svizzera italiana. We collaborate with Prof. Ernst Wit on the development of graphical modelling approaches and their application to genomics.</li>
  <li>Scienze Economiche, Aziendali e Statistiche, Università degli Studi di Palermo. We collaborate with Prof. Luigi Augugliaro on computational statistical approaches for regularized inference.</li>
</ul>

<h3 id="thesis-proposals-1">Thesis proposals</h3>

<p>Please contact <a href="https://webapps.unitn.it/du/it/Persona/PER0222143/Curriculum">Prof. Vinciotti</a> if you are interested in research projects regarding possible topics for your master thesis or PhD in statistics.</p>
